compiler_test.go

```diff
@@ -0,0 +1,23 @@
+package jsonschema
+
+import "testing"
+
+func TestCompiler_AddResourceJSON(t *testing.T) {
+	c := NewCompiler()
+	err := c.AddResourceJSON("main.json", map[string]any{
+		"type": "object",
+		"properties": map[string]any{
+			"id": map[string]any{
+				"type":   "string",
+				"format": "uuid",
```

Comment:
format checks are not enabled by default in draft2020. change the test case to validate just `type` keyword to make things simple.

Reply:
applied